docs/installation_prod.md

```diff
@@ -195,18 +195,18 @@ server {
 
   server_name your.domain;
 
-  # forward traffic to adhocracy-plus
+  # forward traffic to kosmo
   location / {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
     proxy_set_header Host $http_host;
     proxy_pass http://127.0.0.1:8000;
```

Comment:
Do I need to change something here? @goapunk

Reply:
no, this should be fine, it just needs to be the same ip:port as used above in the `ExecStart` line for gunicorn.